src/swap/hooks/useFromTo.ts

```diff
@@ -39,6 +43,7 @@ export const useFromTo = (address?: Address) => {
     loading: toLoading,
     setLoading: setToLoading,
     error: toTokenBalanceError,
+    response: toTokenResponse,
```

Comment:
One more open question.

We say this is a `response`. But what actually we are saying this is a `response` to an Hook related to `UseBalanceReturnType`.

So in reality this is a balance object that helps us `fetching native currency or token balance`.

Mmmmmmm so `response` as name is not actually correct.

It feels more a `balanceSomething`. mmmmm I mean this is the return type https://wagmi.sh/react/api/hooks/useBalance#return-type .

Reply:
it can also be [useReadContractReturnType](https://wagmi.sh/react/api/hooks/useReadContract#return-type) which isn't necessary a balance. I think the `response` convention is from [Tanstack mutations](https://tanstack.com/query/v4/docs/framework/react/guides/updates-from-mutation-responses)

Reply:
but happy to call this something like `balanceResponse` since we only use it for balances (ETH or tokens)

Reply:
regardless I think since this is only used internally for now we can always rename it later on without introducing a breaking change

Reply:
IMO refetching is the only useful field exposed from this object but not that familiar with Tanstack, maybe @alissacrane-cb can provide more input

Reply:
> regardless since this is only used internally
if somethings it's hard to understand, than for us it's hard to refactor. Especially as we have so many folks working on the same files.

Let's go ahead with `balanceResponse`.